spec/features/wrong_word_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe 'test with wrong word' do
+
+  it 'wrong word' do
+    Card.create(original_text: "word", translated_text: "slovo", review_date:"01.01.1900")
+    visit root_path
+    fill_in 'user_answer', with: 'wrong word'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.